### ISSUE TITLE
Remove other search precision levels

### ISSUE THREAD

**Issue by taooceros** [Flow-Launcher/Flow.Launcher#528]
### Discussed in https://github.com/Flow-Launcher/Flow.Launcher/discussions/517

<div type='discussions-op-text'>

<sup>Originally posted by **jjw24** April 25, 2020</sup>
Unless I am wrong, other search precision levels- None, Low are not used much by user base. This assumption is that precision level saving is broken and no one has put forward an issue about it in the JJW24 fork.

![image](https://user-images.githubusercontent.com/26427004/80270083-7fa8c600-86f8-11ea-8a2f-d15252195206.png)

Removing others can help simplify logic and focus on fine tuning the search for one level going forward.</div>

**Comment by skjerns** [comment]
Wrong! I use `None`, as otherwise some of my results are not being shown properly, e.g. programms using Umlauts or searching for special chars. E.g. when I want to open Notepad++ I type `++`, but this only shows the app when setting precision to "None" or low

**Comment by jjw24** [comment]
Can you give me some examples of the searches you are not getting with regular

**Comment by skjerns** [comment]
I'm using 1.8.1, I just checked the forum as each version update resets my settings for the precision value.

I can see if I can remember more cases, but for that I need to turn it off and wait a few days

**Comment by taooceros** [comment]
Probably the storage of SearchPrecision Score contains an issue. Will take a look on that soon.

**Comment by pc223** [comment]
Yeah I never use this feature, I always use the default. We should strive for a good default first and options later.

If User fails to find a item with a reasonable query, it should be consider bug.

**Comment by jjw24** [comment]
> ... We should strive for a good default first and options later.
> 
> If User fails to find a item with a reasonable query, it should be consider bug.

agree, absolutely.

**Comment by skjerns** [comment]
I agree!

my problem is, that `++` doesn't ever show up Notepad if setting to `regular`, so I can't really increase the frequency and "train" it. Strangely enough, the c++ compiler shows up! I have the exact same problem on different machines, so it's not just a local problem

**Comment by pc223** [comment]
> my problem is, that `++` doesn't ever show up Notepad if setting to `regular`

Hm, weird, I tried a fresh install of 1.8.1, seems `notepad++` still show in the bottom (with regular on)

**Comment by jjw24** [comment]
> Hm, weird, I tried a fresh install of 1.8.1, seems notepad++ still show in the bottom (with regular on)

On fresh install, if you query 'notepad++' it will show somewhere on the list for sure, but you will never get it with '++' query, and consequently you cant as @skjerns mentions, train it with frequency to show up higher when query '++'

**Comment by pc223** [comment]
> > Hm, weird, I tried a fresh install of 1.8.1, seems notepad++ still show in the bottom (with regular on)
> 
> On fresh install, if you query 'notepad++' it will show somewhere on the list for sure, but you will never get it with '++' query, and consequently you cant as @skjerns mentions, train it with frequency to show up higher when query '++'

No, I tried with `++` not `notepad++` 😄 But i use notepad++ portable so maybe different there.

**Comment by skjerns** [comment]
Yes, I'm using the regularly installed version v7.9.5 64bit. Updated to Flow 1.8.2, problem still persists :-/

**Comment by taooceros** [comment]
I don't think training it helps, because the score bump is done after the result filtering.

**Comment by jjw24** [comment]
if '++' brings up notepad++, then training would help bump it up, unless i am not remembering the design right?

**Comment by taooceros** [comment]
Yeah, but @skjerns cannot find notepad++ with ++. We can optimize it by let special character bumps score for acronym search, but not include it in score panelty.

**Comment by jjw24** [comment]
yeah think if it's purely special symbol characters or numerals only in the query, we can bump for acronym and not include in score penalty.

**Comment by pc223** [comment]
Agreed, `++` for `notepad++` is a reasonable query, I think we should support this use case by default 👍

**Comment by onesounds** [comment]
Can't we just get rid of this? I've never felt like this function works properly.
